fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalItemTags.java

```diff
@@ -0,0 +1,177 @@
+	public static final TagKey<Item> UNCOLORED_BLOCKS_SANDSTONE = register("sandstone/uncolored_blocks");
+	public static final TagKey<Item> UNCOLORED_SLABS_SANDSTONE = register("sandstone/uncolored_slabs");
+	public static final TagKey<Item> UNCOLORED_STAIRS_SANDSTONE = register("sandstone/uncolored_stairs");
+
+	// Dyes
+	public static final TagKey<Item> DYES = register("dyes");
+	public static final TagKey<Item> BLACK_DYES = register("dyes/black");
+	public static final TagKey<Item> BLUE_DYES = register("dyes/blue");
+	public static final TagKey<Item> BROWN_DYES = register("dyes/brown");
+	public static final TagKey<Item> CYAN_DYES = register("dyes/cyan");
+	public static final TagKey<Item> GRAY_DYES = register("dyes/gray");
+	public static final TagKey<Item> GREEN_DYES = register("dyes/green");
+	public static final TagKey<Item> LIGHT_BLUE_DYES = register("dyes/light_blue");
+	public static final TagKey<Item> LIGHT_GRAY_DYES = register("dyes/light_gray");
+	public static final TagKey<Item> LIME_DYES = register("dyes/lime");
+	public static final TagKey<Item> MAGENTA_DYES = register("dyes/magenta");
+	public static final TagKey<Item> ORANGE_DYES = register("dyes/orange");
+	public static final TagKey<Item> PINK_DYES = register("dyes/pink");
+	public static final TagKey<Item> PURPLE_DYES = register("dyes/purple");
+	public static final TagKey<Item> RED_DYES = register("dyes/red");
+	public static final TagKey<Item> WHITE_DYES = register("dyes/white");
+	public static final TagKey<Item> YELLOW_DYES = register("dyes/yellow");
+
+	// Items created with dyes
+	public static final TagKey<Item> DYED = register("dyed");
+	public static final TagKey<Item> BLACK_DYED = register("dyed/black");
+	public static final TagKey<Item> BLUE_DYED = register("dyed/blue");
+	public static final TagKey<Item> BROWN_DYED = register("dyed/brown");
+	public static final TagKey<Item> CYAN_DYED = register("dyed/cyan");
+	public static final TagKey<Item> GRAY_DYED = register("dyed/gray");
+	public static final TagKey<Item> GREEN_DYED = register("dyed/green");
+	public static final TagKey<Item> LIGHT_BLUE_DYED = register("dyed/light_blue");
+	public static final TagKey<Item> LIGHT_GRAY_DYED = register("dyed/light_gray");
+	public static final TagKey<Item> LIME_DYED = register("dyed/lime");
+	public static final TagKey<Item> MAGENTA_DYED = register("dyed/magenta");
+	public static final TagKey<Item> ORANGE_DYED = register("dyed/orange");
+	public static final TagKey<Item> PINK_DYED = register("dyed/pink");
+	public static final TagKey<Item> PURPLE_DYED = register("dyed/purple");
+	public static final TagKey<Item> RED_DYED = register("dyed/red");
+	public static final TagKey<Item> WHITE_DYED = register("dyed/white");
+	public static final TagKey<Item> YELLOW_DYED = register("dyed/yellow");
+
+	// Other
+	public static final TagKey<Item> STRINGS = register("strings");
+	public static final TagKey<Item> RODS = register("rods");
```

Comment:
Question: is a fishing rod a rod?

Reply:
No as following what forge is using for these rod tags, it's for sticks and blaze rods. And then mods put in their iron rod and other stick-like stuff. Fishing rod is a tool so it falls under tools tag. I don't like the name rod but it seems to be the best name for stick-like items to also cover metal rods that many mods are adding and using. Likely for using as a resource for stuff. Unless you feel fishing rod should be a wooden rod and usable in places like recipes that takes wooden sticks which might be weird

Reply:
While sticks are typically wooden, they can be of any material.

Regardless of name, this should be clarified with javadoc

Reply:
Javadoc added now to be clear it is mainly for recipes.